libs/libxx/Kconfig

```diff
@@ -22,6 +22,14 @@ config HAVE_CXX
 
 if HAVE_CXX
 
+config HAVE_CXXINITIALIZE
+	bool "Have C++ initialization"
+	default n
+	---help---
+		The platform-specific logic includes support for initialization
+		of static C++ instances for this architecture and for the selected
+		toolchain (via up_cxxinitialize()).
```

Comment:
@xiaoxiang781216 up_cxxinitialize() should be removed.  cxx_initialize() is the only system level C++ initializer in the FLAT build.

Reply:
Here: apache/incubator-nuttx-apps#316